app/views/miq_ae_customization/_dialog_sample.html.haml

```diff
@@ -42,7 +42,8 @@
                         - date_val = field.value ? field.value.split(" ") : ["#{t.month}/#{t.day}/#{t.year}"]
                         = datepicker_input_tag("miq_date_1", h(date_val[0]),
                           :class    => "css1",
-                          :readonly => "true")
+                          :readonly => "true",
+                          :data_date_start => field.show_past_dates ? Time.zone.now.strftime("%Y, %m, %d") : "")
```

Comment:
I'd say you want to turn the condition around. I.e. if `field.show_past_dates` is true, you don't want to be setting
`data_date_start`. Right now it would do the exact opposite of what you want to achieve.